tests/test_adjointA.py

```diff
@@ -60,16 +61,15 @@ def test_acoustic(dimensions, time_order, space_order):
         error("Unknown dimension size. `dimensions` parameter"
               "must be a tuple of either size 2 or 3.")
 
-    model = IGrid(origin, spacing, true_vp)
-    # Define seismic data.
+    # Define seismic data
+    model = Model(origin, spacing, true_vp, nbpml=nbpml)
     data = IShot()
     src = IShot()
 
     f0 = .010
+    dt = model.critical_dt
     if time_order == 4:
-        dt = 1.73 * model.get_critical_dt()
-    else:
-        dt = model.get_critical_dt()
+        dt *= 1.73
```

Comment:
Same as the previous comment. This magic number could probably go away as well, in the same step.